src/sentry/eventtypes/error.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import annotations
```

Comment:
This hasn't been mainlined, in fact it never will, so it's still needed for deferred annotations.